### ISSUE TITLE
Flatpak, AppImage

### ISSUE THREAD

**Issue by definitelynobody** [franciscoBSalgueiro/en-croissant#148]
### Describe the feature

It would be nice to have Flatpak or AppImage available for Linux.

**Comment by franciscoBSalgueiro** [comment]
Yes, there's an [AppImage](https://github.com/franciscoBSalgueiro/en-croissant/releases/download/v0.9.0/en-croissant_0.9.0_amd64.AppImage) available on the github release page, I can add it as another option on the website.

Flatpaks are dependent on this Tauri issue.
https://github.com/tauri-apps/tauri/issues/3619

**Comment by franciscoBSalgueiro** [comment]
> You may want to bundle libthai into the AppImage. After installing it as external dependency, the app starts.

That's an open issue in tauri https://github.com/tauri-apps/tauri/issues/4930

**Comment by Romaoplays** [comment]
Tauri has recently added documentation for packaging as Flatpak [https://v2.tauri.app/distribute/flatpak/](https://v2.tauri.app/distribute/flatpak/) as referenced in [issues/3619#issuecomment-2298547576](https://github.com/tauri-apps/tauri/issues/3619#issuecomment-2298547576)

**Comment by franciscoBSalgueiro** [comment]
I'm planning to move to v2 once it gets released. It's currently still in RC

**Comment by SamuelBorn** [comment]
As of [this pull request](https://github.com/franciscoBSalgueiro/en-croissant/pull/407), this project has been updated to Tauri v2.

This means the [Flatpak distribution guide](https://v2.tauri.app/distribute/flatpak/) by @Romaoplays should now be applicable.

It would be amazing to see one of my favorite pieces of software easily available for download on Flathub!

**Comment by PlatonB** [comment]
CC @razzeee ?

**Comment by outusuke** [comment]
https://github.com/outusuke/com.encroissant
